src/protocol/libp2p/kademlia/config.rs

```diff
@@ -52,6 +52,9 @@ const PROTOCOL_NAME: &str = "/ipfs/kad/1.0.0";
 /// Kademlia replication factor.
 const REPLICATION_FACTOR: usize = 20usize;
 
+/// Kademlia maximum message size.
+const DEFAULT_MAX_MESSAGE_SIZE: usize = 1024 * 1024;
```

Comment:
How big are these messages usually? Maybe makes sense to be more conservative? Or is 1MB realistic?

Reply:
The maximum record from the DHT of Kusama is 1827 bytes, I would say capping this at 8 KiB should be enough for most use-cases. I don't expect the bootnodes on DHT to consume much more resources than this either